### PR TITLE
Add TodayCommand

### PR DESCRIPTION
**Pull request by wang-h-z**
Added TodayCommand feature which allows users to filter their clients based on whether they have an appointment `today`.

`Today` is determined by the Java DateTime objects.

### CODE REVIEW

**Review by lrongyi — COMMENTED** [review]
LGTM, better improve the code coverage

File: src/main/java/seedu/address/logic/commands/AddListingCommand.java
Comment (seandias) [inline src/main/java/seedu/address/logic/commands/AddListingCommand.java:1]: LGTM

File: src/main/java/seedu/address/logic/commands/EditListingCommand.java
Comment (seandias) [inline src/main/java/seedu/address/logic/commands/EditListingCommand.java:1]: LGTM

**Review by seandias — COMMENTED** [review]
LGTMMMMM

**Pull request merged**